### DISCUSSION TITLE
More resources

### DISCUSSION THREAD

**Discussion by KalobTaulien** [CAWSCIT/ca-meeting-finder-issue-tracker#19] [Feedback]
A couple people immediately compared the initial release of the Android app to some other 12 step apps, stating that they would like things such as:
* The 12 Steps reading
* The 12 Traditions reading 
* A daily reflection 

These are not currently being planned, as far as I know, but I thought I would log the feedback anyway.

**Comment by enderv** [comment]
This has come up before and previous decision was direct them to world site. We would need to worry about making sure correct loc and such if we did in app.

For daily reflection believe it was copyright issue or some such.
